### PR TITLE
Fix grammar in Readme and PR template

### PR DESCRIPTION
**Pull request by HappyTetrahedron**
Original contribution in https://github.com/appuio/component-openshift4-monitoring/pull/182

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: {{ cookiecutter.slug }}/README.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
 
 ## Documentation
 
 The rendered documentation for this component is available on the [Commodore Components Hub](https://hub.syn.tools/{{ cookiecutter.slug }}).
 
 Documentation for this component is written using [Asciidoc][asciidoc] and [Antora][antora].
-It is located in the [docs/](docs) folder.
-The [Divio documentation structure](https://documentation.divio.com/) is used to organize its content.
+It can be found in the [`docs`](docs) folder.
+The [Divio documentation structure](https://documentation.divio.com/) organizes its content.
```
Comment (simu) [inline {{ cookiecutter.slug }}/README.md:14]: IMO this change makes the sentence even more awkward

I'd reword this as something like

```suggestion
We use the [Divio documentation structure](https://documentation.divio.com/) to organize our documentation.
```

**Review by simu — APPROVED** [review]

File: {{ cookiecutter.slug }}/README.md (hunk 1 above)
Comment (3kh0) [reply {{ cookiecutter.slug }}/README.md:14]: Yes, that does sounds a bit better. 🙃

**Review by 3kh0 — APPROVED** [review]
